src/field/custom_input.jsx

```diff
@@ -31,9 +41,21 @@ const CustomInput = ({ iconUrl, model, name, placeholder, type, validator, value
           {...props}
         />
       );
+    case 'info':
+      return (
```

Comment:
Introducing new TextInfo field

Reply:
I would think textinfo or text is more appropriate.